geode-core/src/main/java/org/apache/geode/internal/metrics/CompositeMeterRegistryFactory.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.internal.metrics;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.composite.CompositeMeterRegistry;
+
+import org.apache.geode.annotations.VisibleForTesting;
+
+/**
+ * Creates {@code CompositeMeterRegistry} and configures commonTags for {@code "ClusterId"} and
+ * {@code "MemberName"}.
```

Comment:
Either add host name to this list or remove any mention of the specific tags. I think "… and configures common tags" suffices.

Reply:
Done!

Reply:
Note: I narrowed CollectingServiceLoader to package-private. If we decide that it's useful enough to move from metrics to some other package, then we can make it public.